app/src/main/res/values-en/strings.xml

```diff
@@ -1192,21 +1192,21 @@ If there are no signs along the whole street which apply for the highlighted sec
     <string name="quest_religion_caodaist">Caodaism</string>
     <string name="quest_religion_chinese_folk">Chinese folk religion</string>
 
-    <string name="quest_road_width_title">"What’s the width of this road here?"</string>
-    <string name="quest_road_width_explanation">"The roadway width from curb to curb: This includes anything on the street surface like on-street parking or cycle lanes but excludes anything beyond it like cycle paths or parking beside the roadway."</string>
+    <string name="quest_road_width_title">"How wide is the road here?"</string>
+    <string name="quest_road_width_explanation">"The roadway width from curb to curb: This includes anything on the street surface, including on-street parking or bike lanes but excluding anything beyond the curb like off-street parking or standalone adjacent bike paths."</string>
```

Comment:
"standalone adjacent bike paths" - there is really not a shorter way to state that? Why not just "bike path"? 

See also https://github.com/streetcomplete/StreetComplete/pull/4030/files#r873592692

Reply:
Maybe the distinction is clearer if there’s a little less focus on cycling infrastructure:

> but excluding anything beyond the curb like off-street parking, sidewalks, sidepaths, or frontage lanes

/ref https://github.com/streetcomplete/StreetComplete/pull/4030#discussion_r874158438

Reply:
The reason I mentioned cycle paths is because cycle paths when they exist are usually the part that is directly adjacent to the parking lane / street.

Reply:
The problem with saying "bike paths" is that it becomes confusing when the bike path is immediately adjacent with minor barriers.

See this example (right-side image):
![image](https://user-images.githubusercontent.com/3254090/168707238-d99e73a5-ce1d-4d87-bc55-1620d079758c.png)

Bike lane and bike path become fuzzy, but I think in this case the bike....lane?... gets included.

So I think maybe we want something like below.  I took the liberty of adding the obviously missing class, sidewalks:

> The roadway width from curb to curb: This includes anything on the street surface, including on-street parking or bike lanes but excluding anything beyond the curb like sidewalks, off-street parking or adjacent bike paths."

Reply:
I'm specifically glossing over terms like "bikeway", "protected bike lane", and "bike path" because they can often be used interchangeably!

See: https://www.hawaiinewsnow.com/2021/08/31/city-adds-bike-lanes-along-ward-avenue-improve-safety-riders/

In this article, **bike lane**, **protected bike lane**, **bike path**, and **bikeway** are ALL used to describe what you see in the right photo!

Reply:
Right, urban planners and cycling advocates have [their own classification system](http://lvbikecoalition.org/wp-content/uploads/2017/12/caltrans-d4-bike-plan_bikeway-classification-brochure_072517.pdf#page=2) that could easily be mapped to OSM’s, but the general public (including presumably a part of the StreetComplete user base) mixes these terms more vigorously than a Class II bikeway at a slip lane.

Reply:
Hmm, for the example photo, I think the width should be like this:

![image](https://user-images.githubusercontent.com/4661658/168797299-ef19a6d7-babe-423f-968f-61097301be82.png)

because the protected bike lane is not part of the street proper anymore because of the permanent barrier. But maybe there are different opinions about that.

In any case, @ZeLonewolf 's suggestion is a definite improvement of the current wording so I will apply that. Thank you!
```suggestion
    <string name="quest_road_width_explanation">"The roadway width from curb to curb: This includes anything on the street surface, including on-street parking or bike lanes but excluding anything beyond the curb like sidewalks, off-street parking or adjacent bike paths.""</string>
```